neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
```

Comment:
Docstring, k čemu je to dobré. 

Tohle vlastně slouží jenom RNN dekodéru. Pokud bude dekodér sequence labeling nebo nedej bože jednoho dne třeba CTC, tak, tak se to nepoužije. Zvážil bych jestli by se to nemělo přestěhovat do adresáře decoders nebo rovnou do `decoders/decoder.py`, který by se pak mohl jmenovat `rnn_decoder.py`.